Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/machine-config-operator
+COPY . .
+RUN WHAT=machine-config-operator ./hack/build-go.sh
+RUN WHAT=machine-config-daemon ./hack/build-go.sh
+RUN WHAT=machine-config-controller ./hack/build-go.sh
+RUN WHAT=machine-config-server ./hack/build-go.sh
+RUN WHAT=setup-etcd-environment ./hack/build-go.sh
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/machine-config-operator/_output/linux/amd64/machine-config-operator /usr/bin
```

Comment:
should this be `/usr/bin/` (with the slash at the end)? same with the rest of the copy commands

Reply:
ahh those got lost in the shuffle - thank you!!

Reply:
done